### PR TITLE
Refresh importance

### PR DESCRIPTION
**Pull request by NemTechCN**

### CODE REVIEW

File: src/core/model/AppWallet.ts [hunk 1]
```diff
@@ -529,4 +529,17 @@ export class AppWallet {
...
         return PublicAccount.createFromPublicKey(this.publicKey, this.networkType)
     }
+
+    async refreshImportance(store: Store<AppState>): Promise<void> {
+      const that = this
+        try {
+            const accountInfo = await new AccountHttp(store.state.account.node)
+                .getAccountInfo(Address.createFromRawAddress(store.state.account.wallet.address))
+                .toPromise()
+            this.importance = accountInfo.importance.compact();
```
Comment (evias) [inline src/core/model/AppWallet.ts]: this should probably reset the *complete account info*, not just the importance. Wasting one getAccountInfo call to just read the importance seems to be off.

File: src/core/model/AppWallet.ts [hunk 2]
```diff
@@ -529,4 +529,17 @@ export class AppWallet {
...
+            that.updateWallet(store);
```
Comment (evias) [inline src/core/model/AppWallet.ts]: what does updateWallet do if it only takes the store?

File: src/views/wallet/wallet-details/WalletDetails.vue [hunk 3]
```diff
@@ -35,6 +35,9 @@
                   <sup>{{(importance+'').length-1}}</sup>
                 </span>
                 <span v-else>0</span>
+                <span class="edit-wallet-name" @click.stop="refreshImportance">
```
Comment (evias) [inline src/views/wallet/wallet-details/WalletDetails.vue]: CSS class is confusing `edit wallet` is not related to `refresh importance` action.

File: src/views/wallet/wallet-details/WalletDetailsTs.ts [hunk 4]
```diff
@@ -108,6 +110,19 @@ export class WalletDetailsTs extends Vue {
...
+    refreshImportance(){
+      const {importanceRefreshTimestamp, wallet} = this
+      const currentTimestamp = new Date().valueOf()
+      if (currentTimestamp - importanceRefreshTimestamp <= 2000) {
+        this.$Notice.destroy()
+        this.$Notice.warning({title: '' + this.$t(Message.REFRESH_TOO_FAST_WARNING)})
+        return
+      }
+        this.importanceRefreshTimestamp = currentTimestamp
+        new AppWallet(wallet).setAccountInfo(this.$store)
```
Comment (evias) [inline src/views/wallet/wallet-details/WalletDetailsTs.ts]: 1) why do you need to instantiate a new instance of `AppWallet` ? 

2) `setAccountInfo` only takes the *store*, maybe this action should be done _in the store_ ?

Comment (decentraliser) [reply src/views/wallet/wallet-details/WalletDetailsTs.ts]: About point 2, this should be done in another PR that rewiews AppWallet (refactor, fix instantiation methods (the store should be passed to the constructor of an AppWallet)

**Pull request closed without merging**